pkgs/development/python-modules/typing-inspect/default.nix

```diff
@@ -16,16 +18,26 @@ buildPythonPackage rec {
     sha256 = "1dzs9a1pr23dhbvmnvms2jv7l7jk26023g5ysf0zvnq8b791s6wg";
   };
 
+  patches = lib.optional (pythonAtLeast "3.9") [
```

Comment:
Can we apply this unconditionally?

Reply:
I don't think so. I saw failures if the patch is applied to Python 3.7/3.8. Let me test again.